app/views/play/modal/PlayHeroesModal.coffee

```diff
@@ -179,66 +179,14 @@ module.exports = class PlayHeroesModal extends ModalView
     return unless hero = @heroes.models[heroIndex]
     @loadHero hero, heroIndex, true
 
-  loadHero: (hero, heroIndex, preloading=false) ->
-    createjs.Ticker.removeEventListener 'tick', stage for stage in _.values @stages
-    createjs.Ticker.framerate = 30  # In case we paused it from being inactive somewhere else
+  loadHero: (hero, _heroIndex, preloading=false) ->
```

Comment:
As a compromise for changing the API I'm demonstrating that _heroIndex isn't used. But maybe I should just chnge API and modify call sites.

Reply:
We should go ahead and remove it if unused.  Call sites will still be misleading unless you look here.